keyboards/order/order.c

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2019 Carlos Rödiger
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "order.h"
+
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+
+	matrix_init_user();
+}
+
+uint8_t selected_layer = 0;
+void encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
This should be set weakly:
```suggestion
__attribute__((weak))
void encoder_update_user(uint8_t index, bool clockwise) {
```

That way, a user can override the function

Reply:
> Also, this looks to be a personal project?
> 
> If so, could this be moved to the handwired folder, where we try to keep handwired and personal projects?

Will it still be available through the configurator? Two friends have stated interest in buying the keyboard but the configurator is a must for them because they aren't too comfortable using the command line (like me)?

Reply:
> Will it still be available through the configurator?

Yes, it will.